sanity/dist/index.html

```diff
@@ -0,0 +1,130 @@
+<!doctype html><html><head><meta charSet="utf-8"/><title>Techo – Sanity</title><meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"/><meta name="robots" content="noindex"/><style>html {background-color: #e4e8ed;}</style><link rel="stylesheet" href="/sanity/static/css/main.css?97523b31dae5df8d781f"/><link rel="subresource" href="/sanity/static/js/vendor.bundle.js?75a1b19ec1d6ce4d4b6c"/><link rel="subresource" href="/sanity/static/js/app.bundle.js?97523b31dae5df8d781f"/><link rel="icon" href="/sanity/static/favicon.ico"/></head><body id="sanityBody"><div id="sanity"><div class="sanity-app-loading-screen"><style type="text/css">
```

Comment:
Im pretty sure we could remove the /dist folder as surely it gets generated when we execute `sanity build` in vercel. 

Their example app has it gitignored

https://github.com/sanity-io/sanity-template-nextjs-ecommerce/tree/a078522b969b48e34a9327918c71eeadb3991e94/studio

Reply:
Dale!